docker/stackbase/webserv/entrypoint.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+# Start nginx
+nginx
+
+# Activacte Stack, setup dax_webserv, and run
+# uwsgi under the lsst UID.
+source /opt/lsst/software/stack/loadLSST.bash
+setup lsst_distrib
+uwsgi --master --processes 40 --threads 1 \
+    --socket /tmp/webserv.sock --uid lsst \
```

Comment:
We should go ahead and keep it  this way for now, but just a note for the future:

We should be able to operate uwsgi in HTTP mode and get rid of nginx in this container completely:
https://uwsgi-docs.readthedocs.io/en/latest/HTTP.html

Reply:
Yeah I was wondering about that as well, but that can be a later commit for sure.  I don't like nginx there, it's just another layer for things to get lost and have to configure!